doc/development-context-aware-applications/v2/how-to-perform-geo-located-queries.md

```diff
@@ -0,0 +1,20 @@
+One very powerful feature in Context Broker GE is the ability to perform
+geo-located queries. You can query entities using the following spatial relationships: 
+
+ * Proximity to a geometry.
+ * Covered by a defined geometry.
+ * Intersected with a reference gometry.
+ * Equality to a geometry.
+ * Not intersected with the geometry.
+
```

Comment:
It would be good to show the georel relationship i.e

proximity to a geometry (`georel=near`)